cfme/base/ui.py

```diff
@@ -1709,8 +1709,9 @@ class import_git(View):     # noqa
     reset_all = VersionPick({
         Version.lowest(): Image(
             './/img[starts-with(@alt, "Reset all components in the following domains:")]'),
-        '5.10': Button(title='Reset all components in the following domains:')
+        '5.10': Button(title='Reset all components in the following domains: RedHat, ManageIQ')
     })
+    reset_title = Text('//*[@id="main_div"]/div[3]/h3[4]')
```

Comment:
Can the locator be more specific ?

Reply:
@sshveta 
I have already tried to make it more specific than existing but I did not find more specific ```xpath``` than this one. Please let me know if you have any solution around it.

Reply:
:+1:

Reply:
@ganeshhubale, isn't there any better xpath ?  
What I don't like in it
1. it uses indexes. So, any slight change of html page may break it.
2. Why do you use full xpath ? As far as I see your view is based on other views. So, I assume you can use relative locator here.